src/Binance/Spot/API/DepositLog.jl

```diff
@@ -0,0 +1,129 @@
+module DepositLog
+
+export DepositLogQuery,
+    DepositLogData,
+    deposit_log
+
+using Serde
+using Dates, NanoDates, TimeZones
+
+using CryptoAPIs.Binance
+using CryptoAPIs: Maybe, APIsRequest
+
+@enum DepositStatus begin
+    PENDING = 0
+    SUCCESS = 1
+    CREDITED_BUT_CANNOT_WITHDRAW = 6
+    WRONG_DEPOSIT = 7
+    WAITING_USER_CONFIRM = 8
+end
+
+Base.@kwdef mutable struct DepositLogQuery <: BinancePrivateQuery
+    coin::Maybe{String} = nothing
+    endTime::Maybe{DateTime} = nothing
+    limit::Maybe{Int64} = 1000
+    offset::Maybe{Int64} = nothing
+    startTime::Maybe{DateTime} = nothing
+    status::Maybe{DepositStatus} = nothing
+    txId::Maybe{String} = nothing
+
+    recvWindow::Maybe{Int64} = nothing
+    timestamp::Maybe{DateTime} = nothing
+    signature::Maybe{String} = nothing
+end
+
+function Serde.SerQuery.ser_type(::Type{<:DepositLogQuery}, x::DepositStatus)::Int64
+    return Int64(x)
+end
+
+struct DepositLogData <: BinanceData
+    address::String
+    addressTag::String
+    amount::Float64
+    coin::String
+    confirmTimes::String
+    id::Int64
+    insertTime::NanoDate
+    network::String
+    status::DepositStatus
+    transferType::Int64
+    txId::Maybe{String}
+    unlockConfirm::Int64
+    walletType::Int64
+end
+
+"""
+    deposit_log(client::BinanceClient, query::DepositLogQuery)
+    deposit_log(client::BinanceClient; kw...)
+
+Fetch deposit history.
+
+[`GET sapi/v1/capital/withdraw/history`](https://binance-docs.github.io/apidocs/spot/en/#deposit-history-supporting-network-user_data)
+
+## Parameters:
+
+| Parameter  | Type          | Required | Description |
+|:-----------|:--------------|:---------|:------------|
+| coin       | String        | false    |             |
+| endTime    | DateTime      | false    |             |
+| limit      | Int64         | false    |             |
```

Comment:
Add default values to description if they're not `nothing`

Reply:
Done